src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <div className="App">
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${person.name}`}
+      </h2>
+      {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+      )}
+      <p className="Person__partner">
+        {person.isMarried
+          ? `${person.partnerName} is my 
+            ${person.sex === 'f'
+            ? 'husband'
+            : 'wife'}`
+          : 'I am not married'
+        }
```

Comment:
try to avoid nested ternary operators, move 
`${person.sex === 'f'
    ? 'husband'
    : 'wife'}``
into variable